libs/application/templates/rental-agreement/src/utils/rentalPeriodUtils.ts

```
@@ -1,122 +1,85 @@
+import { parse, format } from 'date-fns'
```

ESLint

[error] 'date-fns' import is restricted from being used.

(no-restricted-imports)